_docs/concepts/policy-and-control/tcp-services.md

```diff
@@ -0,0 +1,80 @@
+---
+title: TCP Services
+overview: Overview of Istio's support for TCP Services, including the attributes available for policy and telemetry.
+              
+order: 50
+
+layout: docs
+type: markdown
+---
+
+This page provides an overview of the support within Istio for TCP services.
+
+## Overview
+
+As of the 0.2 release, Istio supports adding TCP services to an Istio Service
```

Comment:
Nit, we have always had support for tcp in Pilot. So, i would scope this sentence to "support for enforcing policies on plain TCP services". Or probaby even get rid of it.

Reply:
See release notes for how we phrased this.